### ISSUE TITLE
paper-header-panel accessibility issues

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-header-panel#11]
For the most part, this is very straightforward. I'm assuming I'm not supposed to be able to use the keyboard to interact with the panels, as they are really just providing information to the user and are not actionable. 

Here are the audit results: 
[Warning] Text elements should have a reasonable contrast ratio (7)

The white on blue isn't bad, but since the font is pretty thin it makes it a little difficult to see. Any chance we could make the white text more bold, or slightly tweak the color of the blue to be darker? 

Windows specific: 
This works fine across screen readers. The only thing I'll mention is that on Firefox with NVDA, the last stop on the page (after Cover) says "blank", as if it's recognizing that something is there (but it's really not).

**Comment by blasten** [comment]
This is all fixed now. In regard to your last point, @cdata commented that that's a NVDA feature.
